tpm2/keydata.go

```diff
@@ -140,6 +141,8 @@ func newKeyData(keyPrivate tpm2.Private, keyPublic *tpm2.Public, importSymSeed t
 // SealedKeyObject corresponds to a sealed key data file.
 type SealedKeyObject struct {
 	data keyData
+
+	k *secboot.KeyData
```

Comment:
is there some way to avoid this? have two types? when reading all the code is quite confusing to have to keep track of when a SeledKeyDataObject has this set or not

Reply:
This is fixed by PR #250 now.